test/ex_sdp/media_test.exs

```diff
@@ -264,12 +264,36 @@ defmodule ExSDP.MediaTest do
         payload_type: 96
       }
 
+      ssrc = %SSRC{
+        id: 12345,
+        attribute: "cname"
+      }
+
+      fmtp = %FMTP{
+        pt: 96
+      }
+
+      msid = %MSID{
+        id: "DycBRAGTwt75ESYihb03FsVWVs8sSdIkhTqN"
+      }
+
```

Comment:
Let's write each map in one line. It will take less space. Also "cname" should have value. It doesn't matter in case of test but cname always has value e.g. `%SSRC{id:12345, attribute: "cname", value: "HPd3XfRHXYUxzfsJ"}`.